src/components/Navbar/Navbar.js

```diff
@@ -24,19 +20,9 @@ class Navbar extends Component {
                                     </Link>
                                 </li>
                             )
-
                         })}
                     </ul>
                 </nav>
-                <div>
-                    <switch>
-                        <div>
-                            <Route exact path="/" component={Home}/>
-                            <Route exact path="/agency" component={Agency}/>
```

Comment:
How nitpicky are our style requirements? In this file, sometimes there are spaces around equals signs in DOM elements and sometimes there aren't, and there aren't spaces before the slashes here

Reply:
let's keep spaces consistent so update these to all follow the same style